systems/common/spack_includes.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+packages:
+  blt:
```

Comment:
@michaelmckinsey1 why is this here?

Reply:
Peter said this constraint should still be used across all systems for blt ([see here](https://spackpm.slack.com/archives/C06ABDD82BF/p1741036610163819?thread_ts=1740428656.070949&cid=C06ABDD82BF))